app/models/user.rb

```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
Remove this because there is no admin feature yet

Reply:
Removed.